language/stdlib/modules/Libra.move

```diff
@@ -692,6 +682,15 @@ module Libra {
         // TODO (dd): explore why I'm getting an error saying has_register_new_currency_privilege assert in
         // code is not covered.
         // aborts_if !Roles::spec_has_register_new_currency_privilege_addr(Signer::spec_address_of(lr_account));
+        include RegisterCurrencyAbortsIf<CoinType>;
+        ensures spec_is_currency<CoinType>();
+        ensures spec_currency_info<CoinType>().total_value == 0;
+    }
+
+    spec schema RegisterCurrencyAbortsIf<CoinType> {
+        lr_account: signer;
+        // TODO (dd): Incomprehensible "function does not abort under this condition"
```

Comment:
This is probably due to a bug in Roles. The definition of `has_register_new_currency_privilege(account)` is `has_libra_root_role(account)` while the definition of `spec_has_register_new_currency_privilege_addr(addr)` is `spec_has_treasury_compliance_role_addr(addr)` instead of `spec_has_libra_root_role_addr(addr)`.